app/Controllers/userController.php

```diff
@@ -211,9 +211,16 @@ public function manageAction() {
 		}
 	}
 
-	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfig = array(), $insertDefaultFeeds = true) {
-		if (!is_array($userConfig)) {
-			$userConfig = array();
+	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfigOverride = array(), $insertDefaultFeeds = true) {
+		$userConfig = array();
+
+		$customUserConfig = @include(join_path(DATA_PATH, 'config-user.custom.php'));
```

Comment:
Wouldn't it be better to do

```
if is_readable then
  $custom = include(etc)
```

Otherwise you're suppressing all errors when really all we (presumably) want to avoid is file not found errors.